RecoTauTag/RecoTau/interface/RecoTauQualityCuts.h

```diff
@@ -20,10 +20,12 @@
  */
 
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
 #include "DataFormats/Candidate/interface/Candidate.h"
 #include "DataFormats/Candidate/interface/CandidateFwd.h"
 #include "DataFormats/TrackReco/interface/Track.h"
 #include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
```

Comment:
@veelken I guess there is no need anymore for this include, right? Please just confirm. I can remove it during migration.

Reply:
@swozniewski  I confirm that this include statement is not necessary anymore. I have removes this line from the code and updated the PR accordingly.

Reply:
thank!